util/storage-samples/storage-owner-sample/src/main/java/org/fidoalliance/fdo/storage/OwnerDbManager.java

```diff
@@ -346,9 +346,9 @@ public void updateMtu(DataSource ds, String field, int mtu) {
         mtu = Const.SERVICE_INFO_MTU_MIN_SIZE;
```

Comment:
The 1st 'if' is redundant because of 2nd 'if'. Unless it's just there for logging purposes, maybe remove the 1st if altogether?